DiveMeets/DiveMeets/Views/Home/Home.swift

```diff
@@ -631,72 +631,81 @@ struct MeetBubbleView: View {
 
 struct HomeColorfulView: View{
     @Environment(\.colorScheme) var currentMode
+    private var screenWidth = UIScreen.main.bounds.width
+    private var screenHeight = UIScreen.main.bounds.height
```

Comment:
```suggestion
    private let screenWidth = UIScreen.main.bounds.width
    private let screenHeight = UIScreen.main.bounds.height
```
We should use let instead here since we don't want those values to be modifiable.